healthyregions/SDOHPlace
# feat: basic Toolkit styling for CMS markdown

## Problem
Styling for Guides markdown is not consistent with other products

For example, the [SDOH Toolkit](https://toolkit.sdohplace.org/07-dissem.html) has custom styling for things like headers, lists, tables, etc

Fixes #326 

## Approach
* feat: ported some styles over to Guides (hopefully without affecting other views like News or Showcases)
    * Basic styling for headers, lists, and accordions (details + summary)
    * Styling for table rows/headers/borders - something is interfering with `font-weight`, and I'm not sure what 🤔  

## How to Test
1. Navigate to https://deploy-preview-327--cheerful-treacle-913a24.netlify.app/guides/public-transit-equity
    * You should see new styling here that more closely matches what we see in the [SDOH Toolkit](https://toolkit.sdohplace.org/04-data.html)
    * Compare with production: https://sdohplace.org/guides/public-transit-equity
